### ISSUE TITLE
Norman not showing as an option on macOS’s login screen?

### ISSUE THREAD

**Issue by handcoding** [deekayen/norman#15]
Try as I might, I can’t seem to get Norman to show up as an option on macOS’s login screen. I’ve followed the steps from the [readme](https://github.com/deekayen/norman/blob/master/mac/README.md) as best as I can:

- ✓ I checked that Norman.bundle is in /Library/Keyboard Layouts/:

![library - keyboard layouts](https://user-images.githubusercontent.com/118319/38438697-dba451a6-39a0-11e8-97fc-8bf2fecf32d3.png)

- ✓ I made sure that Norman is listed under System Preferences → Keyboard → Input Sources
- ✓ And I checked the option for “Show Input menu in menu bar”

![system preferences - keyboard - input sources](https://user-images.githubusercontent.com/118319/38438786-258b2b1e-39a1-11e8-81f2-bc6688f5e549.png)

- ✓ I checked the option for “Show Input menu in login window” under System Preferences → Users and Groups → Login Options

![system preferences - login options](https://user-images.githubusercontent.com/118319/38438792-2b6f1f54-39a1-11e8-8792-f1a54dcedff1.png)

When I go to log in, there’s an Input menu, but I don’t see “Norman” in the main list of input sources or under the list of “Other Input Sources”:

![macos login screen - input selection](https://user-images.githubusercontent.com/118319/38438840-50313aac-39a1-11e8-954f-e249cc75f433.jpg)

![macos login screen - input selection - other](https://user-images.githubusercontent.com/118319/38438846-53704f3c-39a1-11e8-86fc-46acefcd215f.jpg)

Would you happen to have any ideas there?

(Just for completeness: I’m seeing this on my MacBook Pro, which is running El Capitan 10.11.6, and I’m also seeing this on my Mac mini, which is running High Sierra 10.13.4.)

P.S. Norman works fine once I log in (and in fact, I’m wrote this with Norman). It only seems to be on the login screen where I’m running into this.

**Comment by deekayen** [comment]
Yes, it used to be there. The Users & Groups preference checkbox was the big key to showing it after it was activated by any user. None of the keyboard layouts from the Norman challenge are showing for me anymore in the login menu.

**Comment by handcoding** [comment]
> I'm not excited about the possibility of having to sign the bundle with a $99/yr developer account.

If it comes to that, I’d be happy to chip in toward that cost if you were to set up a GoFundMe or something. (It’d be the least that I could do to thank you for all your hard work in creating this.)

**Comment by deekayen** [comment]
I don't yet know if that's the fix or not.
